crates/net/src/websocket/futures.rs

```diff
@@ -33,6 +33,7 @@ use futures_channel::mpsc;
 use futures_core::{ready, Stream};
 use futures_sink::Sink;
 use gloo_utils::errors::JsError;
+use gloo_utils::format::JsValueSerdeExt;
```

Comment:
Can you remove this if it's not needed?

Reply:
It's used on line 110 though.

Reply:
Ah, it's unused unless som feature is enabled. Fixed.